src/include/migraphx/inline_subgraph.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef MIGRAPHX_GUARD_RTGLIB_INLINE_SUBGRAPH_HPP
+#define MIGRAPHX_GUARD_RTGLIB_INLINE_SUBGRAPH_HPP
+
+#include <string>
+#include <migraphx/instruction_ref.hpp>
+#include <migraphx/config.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+
+struct module;
+
+struct inline_subgraph
```

Comment:
I think the pass should be called `inline_module`, to be consistent with terminology.

Reply:
fixed